openapi_python_client/parser/responses.py

```diff
@@ -22,6 +22,7 @@ class Response:
     "application/json": "response.json()",
     "application/octet-stream": "response.content",
     "text/html": "response.text",
+    "text/yaml": "yaml.safe_load(response.text.encode('utf-8'))",
```

Comment:
That's a lot of logic to put in the value - is it possible to encapsulate most of that somewhere into a function that accepts `response`?

```suggestion
    "text/yaml": "to_yaml(response)",
```

Reply:
Broke this down on the latest commit